tests/tensorflow/test_printing.py

```diff
@@ -94,51 +94,34 @@ def test_shape_index():
 
     z = tf.matmul(A, tf.add(y, y, name='x_p_y'), name='A_dot')
 
-    
+
     range_out = io.StringIO()
     with redirect_stdout(range_out):        
         tf_dprint(z, depth_index=range(3, 6))
 
     expected_out = textwrap.dedent('''
-    .
-    .
-    .
-    |  |  |  Op(Add)\t"x_p_y"
-    |  |  |  |  Tensor(Mul):0,\tshape=[None, 1]\t"y:0"
-    |  |  |  |  |  Op(Mul)\t"y"
-    .
-    .
-    .
-    |  |  |  |  Tensor(Mul):0,\tshape=[None, 1]\t"y:0"
-    |  |  |  |  |  Op(Mul)\t"y"
-    .
-    .
-    .
+    ... Op(Add)\t"x_p_y"
+    ... |  Tensor(Mul):0,\tshape=[None, 1]\t"y:0"
+    ... |  |  Op(Mul)\t"y"
+    ... |  Tensor(Mul):0,\tshape=[None, 1]\t"y:0"
+    ... |  |  Op(Mul)\t"y"
     ''')
 
+
     assert range_out.getvalue() == expected_out.lstrip()
 
     list_out = io.StringIO()
     with redirect_stdout(list_out):        
         tf_dprint(z, depth_index=[3, 4, 6])
 
     expected_out = textwrap.dedent('''
-    .
-    .
-    .
-    |  |  |  Op(Add)\t"x_p_y"
-    |  |  |  |  Tensor(Mul):0,\tshape=[None, 1]\t"y:0"
-    .
-    .
-    .
-    |  |  |  |  |  |  Tensor(Const):0,\tshape=[]\t"y/x:0"
-    |  |  |  |  |  |  Tensor(Placeholder):0,\tshape=[None, 1]\t"x:0"
-    |  |  |  |  Tensor(Mul):0,\tshape=[None, 1]\t"y:0"
-    .
-    .
-    .
-    |  |  |  |  |  |  Tensor(Const):0,\tshape=[]\t"y/x:0"
-    |  |  |  |  |  |  Tensor(Placeholder):0,\tshape=[None, 1]\t"x:0"
+    ... Op(Add)\t"x_p_y"
+    ... |  Tensor(Mul):0,\tshape=[None, 1]\t"y:0"
+    ... |  |  |  Tensor(Const):0,\tshape=[]\t"y/x:0"
+    ... |  |  |  Tensor(Placeholder):0,\tshape=[None, 1]\t"x:0"
```

Comment:
You're missing an indicator for the ommitted depths here.

Reply:
Fixed in latest commit.